test_conformance/math_brute_force/reference_math.h

```diff
@@ -205,7 +204,8 @@ long double reference_acoshl(long double x);
 long double reference_asinhl(long double x);
 long double reference_atanhl(long double x);
 long double reference_cbrtl(long double x);
-long double reference_copysignl(long double x, long double y);
+double reference_copysignl(double x, double y);
+long double reference_copysignll(long double x, long double y);
```

Comment:
This part of the change is counter-intuitive: the reference_math function names follow the suffix convention of the C standard library math functions where a single `l` suffix means `long double`; but your change breaks that for `copysign`.

Instead I would suggest modifying the entry for `copysign` in `functionList`, to explicitly state the reference functions (replacing the `ENTRY` macro for `copysign`).

Reply:
Here too I was just following what was already done for `copysign`: all the `reference_*` functions for `float` take `double` arguments, except for `copysign`. I agree that it's counter-intuitive though, it shouldn't be too much work to clean that up both for the `float` and for the `double` version, so will make that change.

Reply:
Done, thanks for the suggestion!

Reply:
Thanks for tidying up the `float` case too!